examples/framework/preferences/minimalEditor/js/minEditor.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one of these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+(function () {
+    "use strict";
+
+    /**
+     * Panel for the auto-pilot preference
+     */
+    fluid.defaults("minEditor.panels.autoPilot", {
```

Comment:
i was thinking more of flyingCar.panels.autoPilot instead of minEditor.panels.autoPilot. We could leave the folder as minEditor if we think there'll be others later or if it is more descriptive. What are you thoughts on all that?